documentation/asciidoc/accessories/camera/libcamera_apps_multicam.adoc

```diff
@@ -0,0 +1,12 @@
+=== Multiple Cameras Usage
+
+Basic support for multiple cameras is available within `libcamera-apps`. Multiple cameras may be attached to a Raspberry Pi in the following ways:
+
+* Two cameras connected directly to a Raspberry Pi Compute Module board, see link:https://www.raspberrypi.com/documentation/computers/compute-module.html#attaching-a-raspberry-pi-camera-module[here] for more details.
+* Two or more cameras attached to a non-compute Raspberry Pi board using a Video Mux board, like link:https://www.arducam.com/product/multi-camera-v2-1-adapter-raspberry-pi/[this] 3rd party product.
+
+In the latter case, only one camera may be used at a time since both cameras are attached to a single Unicam port. For the former, both cameras can run simultaneously.
```

Comment:
Just out of edge-case-curiosity, what happens if you connect one of those Video Mux boards to a Compute Module?

Reply:
It ought to work fine.  So you could have > 2 cameras attached to a CM4, but only ever use up-to 2 simultaneously.